jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -54,14 +54,28 @@ Your self-managed instance must already contain at least one project. The author
 * A link:https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html[personal access token]. This token must have the `api` scope.  
 
 * Your instance's SSH key fingerprint. You can copy this from your instance's `known_hosts` file, which is located by default in the `~/.ssh/` directory.
+  ** To get this SSH key, use your self-managed instance URL and run `ssh-keyscan gitlab.com/yourInstanceURL`. The output should look something like:
```

Comment:
@sebastian-lerner The instance URL would look something like `example.circleci.com`, correct? Is it OK to change the URL example below as well?

Reply:
i think itll look like gitlab.example.com because it's a gitlab self-managed instance?